README.md

```diff
@@ -286,13 +289,13 @@ under a category in libcxx, or running a single test in `std` and `tr1`.
 These examples assume that your current directory is `C:\Dev\STL\out\x64`.
 
 * This command will run all of the test suites with verbose output.
-  + `ctest -V`
-* This command will also run all of the test suites.
+  + `ninja test extra-asan-tests`

```

Comment:
This is missing `-k 0` (as seen in `azure-devops/run-tests.yml`), so presumably it will stop if the normal tests fail. This is unlike the following stl-lit command, which will keep going. I see several options here:

* Mention this behavioral difference.
* Avoid this behavioral difference by adding `-k 0` here.
* Avoid this behavioral difference by adding an `all-tests` target that runs both normal and ASan.
  + Running `-Dnotags=ASAN` followed by `-Dtags=ASAN` serially is more complicated and slightly less efficient than entirely omitting the tags options.
* My recommendation: Change *all* of these listed commands to omit ASan (i.e. mention only the normal test target here, and add `-Dnotags=ASAN` to all stl-lit invocations). This aligns with our PR checks and our normal development practice.

Reply:
Resolved by removing all mention of running the custom test targets.